infrasim/dae.py

```diff
@@ -0,0 +1,628 @@
+import json
+import re
+
+"""
+  Note:
+  DAEProcessHelper is the entry to this module.
+  All other classes, including Const, Link, Enclosure, Drv, Expander and
+  DAEController,  are used to store internal data for building topology.
+
```

Comment:
🛠 a small example of the dae file here will help

Reply:
That's helpful. We will put an example in wiki page.